website/client/components/static/faq.vue

```diff
@@ -69,8 +95,13 @@
       };
     },
     methods: {
-      setActivePage (page) {
-        this.pageState[page] = !this.pageState[page];
+      isVisible (heading) {
+        return this.visible && this.visible === heading;
+      },
+      handleClick (e) {
+        if (!e) return;
+        const heading = e.originalTarget.nextElementSibling.id;
```

Comment:
`originalTarget` isn't available on every browser, here it would be better to use `target`